controller.rst

```diff
@@ -391,128 +391,10 @@ Request object.
    single: Controller; The session
    single: Session
 
-.. _session-intro:
-
 Managing the Session
 --------------------
 
-Symfony provides a session object that you can use to store information
-about the user between requests. Session is enabled by default, but will only be
-started if you read or write from it.
-
-Session storage and other configuration can be controlled under the
-:ref:`framework.session configuration <config-framework-session>` in
-``config/packages/framework.yaml``.
-
-To get the session, add an argument and type-hint it with
-:class:`Symfony\\Component\\HttpFoundation\\Session\\SessionInterface`::
-
-    use Symfony\Component\HttpFoundation\Response;
-    use Symfony\Component\HttpFoundation\Session\SessionInterface;
-    // ...
-
-    public function index(SessionInterface $session): Response
-    {
-        // stores an attribute for reuse during a later user request
-        $session->set('foo', 'bar');
-
-        // gets the attribute set by another controller in another request
-        $foobar = $session->get('foobar');
-
-        // uses a default value if the attribute doesn't exist
-        $filters = $session->get('filters', []);
-
-        // ...
-    }
-
-Stored attributes remain in the session for the remainder of that user's session.
-
-For more info, see :doc:`/session`.
-
-.. index::
-   single: Session; Flash messages
-
-.. _flash-messages:
-
-Flash Messages
```

Comment:
I think we should keep some of this section in the controller guide (and linking the session docs for the big explanation). Flash messages are useful enough to deserve some special attention here (and also to showcase the `AbstractController::addFlash()` method).

Reply:
I've reverted this partially.